app/models/gem_name_reservation.rb

```diff
@@ -0,0 +1,5 @@
+class GemNameReservation < ApplicationRecord
```

Comment:
We should validate uniqueness of the reservations?

Reply:
added